src/components/FwbCard/composables/useCardClasses.ts

```
@@ -13,17 +14,20 @@
 } {
   const cardClasses = computed(() => {
     let computedClasses = ''
+    
```

ESLint

[error] Trailing spaces not allowed.

(@stylistic/no-trailing-spaces)